src/components/Main/Main.test.js

```diff
@@ -0,0 +1,74 @@
+import { toBeInTheDocument } from "@testing-library/jest-dom/dist/matchers";
+import {
+  findByAltText,
+  getByAltText,
+  getByText,
+  render,
+  screen,
+  waitFor,
+} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import moment from "moment";
+import Main from "./Main";
+
+describe("check components should render", () => {
+  test("check the datepicker exists", () => {
+    render(<Main />);
+    const today = moment().format("YYYY-MM-DD");
+
+    const textElement = screen.getByDisplayValue(today);
+    expect(textElement).toBeInTheDocument();
+  });
```

Comment:
Besides ensuring only date is present in our window, we can check that **mockedImageOfTheDayRendered** fn rendered and passed too for acceptance criteria 1
```suggestion
  test("When the user enters the app, the app should show the Picture of the Day.", async () => {
    render(<Main />);
    const today = moment().format("YYYY-MM-DD");
    const textElement = await waitFor(() => screen.getByDisplayValue(today));
    expect(textElement).toBeInTheDocument();
    mockedImageOfTheDayRendered();
  });
  });
```